sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/redis/passwordless/RedisUrlSyntaxException.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.redis.passwordless;
+
+class RedisUrlSyntaxException extends RuntimeException {
```

Comment:
Does Spring Boot provide this exception?

Reply:
It's package level [exception](https://github.com/spring-projects/spring-boot/blob/29b7c121fb7e173f18dabc998668b8d5d31de936/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/RedisUrlSyntaxException.java#L24) in spring boot.